src/afl-fuzz-state.c

```diff
@@ -496,6 +496,13 @@ void read_afl_environment(afl_state_t *afl, char **envp) {
             afl->max_length =
                 atoi((u8 *)get_afl_env(afl_environment_variables[i]));
 
+          }else if (!strncmp(env, "AFL_PIZZA_MODE",
```

Comment:
Please run `make code-format`

Reply:
not even drunk kinds wrote `}else if` ziocan @CarloMara